kratos.gid/apps/DEM/write/writeMDPA_Parts.tcl

```diff
@@ -304,6 +311,181 @@ proc DEM::write::writeVelocityMeshes { } {
 }
 
 
+
+proc DEM::write::writeDEMConditionMeshes2D { } {
```

Comment:
Are we sure we need to duplicate all this code? They look very similar (3d and 2d). What's the problem if we write the same for both cases? We could just adjust the tree, then write the default value for z component in 2D (0.0).